frontend/src/components/details/person.svelte

```diff
@@ -1,23 +1,24 @@
 <script lang="ts">
-    import {routeToPage} from "../utils";
+    import {routeToPage} from "../../utils";
 
-    export let imgUrl: string;
-    export let media;
-    export let showButtonAmount: number;
-    export let castItemStartAmount: number;
+	const LOW_RES_IMG_URL: string = 'https://image.tmdb.org/t/p/w500/';
+	const SHOW_BUTTON_AMOUNT: number = 18;
+	const CAST_ITEM_START_AMOUNT: number = 9;
+
+    export let cast;
 
     let castItemAmount: number = 9;
 </script>
 
-{#if media.cast.length !== 0}
+{#if cast.length !== 0}
```

Comment:
Did we test if we could do a `#if cast` or `#if cast.length`?

Reply:
True i changed it